ksqldb-engine/src/main/java/io/confluent/ksql/topic/TopicCreateInjector.java

```diff
@@ -50,7 +53,7 @@
  *
  * @see TopicProperties.Builder
  */
-public class TopicCreateInjector implements Injector {
+public class TopicCreateInjector implements InjectorWithSideEffects {
```

Comment:
I assume that `DistributingExecutor` is the only code path that is susceptible to the problem.